packages/atomic/src/components/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -28,6 +28,7 @@ export class AtomicResultImage {
     );
 
     if (typeof url !== 'string') {
+      console.error(`Expected "${this.field}" to be a text field.`, this.host);
```

Comment:
Would be better to use a common logger instance generally where the level could be adjusted (but i'm also fine with just logging directly, since errors should always be displayed IMO)